sanitize

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
this isn't for "activate", it's for an as-yet unimplemented "ping" (derived from canonical-poke) so leaving it pending some decisions there. using the logs and being able to demonstrate how IP information is immediately removed before any further analysis is, I think, why this script exists and is publicly visible.